profile_bluesky/startup/instrument/plans/energy_plans.py

```diff
@@ -0,0 +1,40 @@
+"""
+Slitscan
+"""
+
+__all__ = ['moveE']
+
+
+from bluesky.plan_stubs import mv,stage
+from ..devices import undulator,mono
+
+def moveE(energy):
```

Comment:
Why not make an energy device as a PvPositioner?

Reply:
This is a great point. The problem here is that as we move energy, we may want the undulator and/or phase plates to also move, and I'm not sure how could this be handled as a PVPositioner. We could create a class that contain these three devices, but that might get messy. The other way I can think of would be to subscribe the undulator and phase retarder to the energy, but since these are different devices, I'm not that would well.